packages/catalog-model/src/validation/CommonValidatorFunctions.ts

```diff
@@ -91,7 +91,7 @@ export class CommonValidatorFunctions {
       typeof value === 'string' &&
       value.length >= 1 &&
       value.length <= 63 &&
-      /^[a-z0-9]+(\-[a-z0-9]+)*$/.test(value)
+      /^(?!\-)[a-z0-9\-]*[a-z0-9]$/.test(value)
```

Comment:
```suggestion
      /^[a-z0-9]+(?:\-+[a-z0-9]+)*$/.test(value)
```

Could we keep the existing structure? prefer that over the negative lookahead

Reply:
Sure